app/src/main/java/org/dhis2/data/service/SyncPresenterImpl.kt

```diff
@@ -34,80 +34,80 @@ import org.hisp.dhis.android.core.program.ProgramType
 import timber.log.Timber
 
 class SyncPresenterImpl(
-    private val d2: D2,
-    private val preferences: PreferenceProvider,
-    private val workManager: WorkManager
+        private val d2: D2,
+        private val preferences: PreferenceProvider,
+        private val workManager: WorkManager
 ) : SyncPresenter {
```

Comment:
Take a look at your code style configuration. It should't add more spaces throughout the file

Reply:
It is done in one of the next PRs